### PR TITLE
Replace health check to prevent high CPU and memory spike

### PR DESCRIPTION
**Pull request by RemCom**
# Proposed Changes

Replace health check to prevent high CPU and memory spike

## Related Issues
https://github.com/hassio-addons/addon-emqx/issues/60

### CODE REVIEW

File: emqx/Dockerfile [hunk 1]
```diff
@@ -28,12 +28,6 @@ RUN \
 COPY rootfs /
 
 WORKDIR /opt/emqx
-HEALTHCHECK \
-    --interval=5s \
-    --retries=5 \
-    --start-period=30s \
-    --timeout=25s \
-    CMD /opt/emqx/bin/emqx_ctl status || exit 1
```
Comment (frenck) [inline emqx/Dockerfile:36]: We should not remove a health check.

I'm fine with removing it, if there is a sane alternative provided.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by RemCom** [comment]
Updated dockerfile with check on dashboard

File: emqx/Dockerfile (hunk 1 above)
Comment (RemCom) [reply emqx/Dockerfile:36]: Updated dockerfile witch check on dashboard

Comment (frenck) [reply emqx/Dockerfile:36]: Nice, that would do I guess 👍

**Review by frenck — APPROVED** [review]
Thanks, @RemCom 👍 

../Frenck

**Pull request merged**